### PR TITLE
Update carbon black deprecated content

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Deprecated content on Endpoint Enrichment - Generic v2 was Updated.
A new version of Block Endpoint - Carbon Black Response V2.1 was created - replaced deprecated command and bug fix for retrieving information for all hostnames. 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml [hunk 1]
```diff
@@ -1198,6 +1198,112 @@ outputs:
...
+- contextPath: CarbonBlackEDR.Sensor.node_id
+  description: The node ID associated with the sensor.
+- contextPath: CarbonBlackEDR.Sensor.cookie
+  description: The cookie associated with the sensor.
+- contextPath: CarbonBlackEDR.Sensor.emet_exploit_action
+  description: The EMET exploit action associated with the sensor.
+- contextPath: CarbonBlackEDR.Sensor.computer_name
+  description: NetBIOS name of this computer.
+- contextPath: CarbonBlackEDR.Sensor.license_expiration
+  description: When the licene of the sensor expires.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml:1273]:
```suggestion
  description: When the license of the sensor expires.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml [hunk 2]
```diff
@@ -1198,6 +1198,112 @@ outputs:
...
+- contextPath: CarbonBlackEDR.Sensor.supports_isolation
+  description: Whther sensor supports isolation.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml:1275]:
```suggestion
  description: Whether the sensor supports isolation.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml [hunk 3]
```diff
@@ -1198,6 +1198,112 @@ outputs:
...
+- contextPath: CarbonBlackEDR.Sensor.parity_host_id
+  description: The ID of the parity host associated with the sensor.
+- contextPath: CarbonBlackEDR.Sensor.supports_2nd_gen_modloads
+  description: Whether the sensor support modload of 2nd generation.
+- contextPath: CarbonBlackEDR.Sensor.network_adapters
+  description: A pipe-delimited list list of IP,MAC pairs for each network interface.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml:1281]:
```suggestion
  description: A pipe-delimited list of IP,MAC pairs for each network interface.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml [hunk 4]
```diff
@@ -1198,6 +1198,112 @@ outputs:
...
+- contextPath: CarbonBlackEDR.Sensor.sensor_health_status
+  description: self-reported health score, from 0 to 100. Higher numbers are better.
+- contextPath: CarbonBlackEDR.Sensor.registration_time
+  description: Time this sensor originally registered in server-local time and zone.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml:1285]:
```suggestion
  description: Time this sensor was originally registered in server-local time and zone.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml [hunk 5]
```diff
@@ -1198,6 +1198,112 @@ outputs:
...
+- contextPath: CarbonBlackEDR.Sensor.restart_queued
+  description: Whether a restart of the sensot is queued.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml:1287]:
```suggestion
  description: Whether a restart of the sensor is queued.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml [hunk 6]
```diff
@@ -1198,6 +1198,112 @@ outputs:
...
+- contextPath: CarbonBlackEDR.Sensor.notes
+  description: The notes associated with the sensor.
+- contextPath: CarbonBlackEDR.Sensor.num_storefiles_bytes
+  description: Number of storefiles bytes associated with the sensor.
+- contextPath: CarbonBlackEDR.Sensor.os_environment_id
+  description: The ID of the os enviroment of the sensor.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml:1293]:
```suggestion
  description: The ID of the OS environment of the sensor.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml [hunk 7]
```diff
@@ -1198,6 +1198,112 @@ outputs:
...
+- contextPath: CarbonBlackEDR.Sensor.shard_id
+  description: The ID of the shard associated with the sensor.
+- contextPath: CarbonBlackEDR.Sensor.boot_id
+  description: A sequential counter of boots since the sensor was installed.
+- contextPath: CarbonBlackEDR.Sensor.last_checkin_time
+  description: Last communication with this computer in server-local time and zone.
+- contextPath: CarbonBlackEDR.Sensor.os_type
+  description: The operating system type of the computer.
+- contextPath: CarbonBlackEDR.Sensor.group_id
+  description: The sensor group id this sensor is assigned to.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml:1303]:
```suggestion
  description: The sensor group ID this sensor is assigned to.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml [hunk 8]
```diff
@@ -1198,6 +1198,112 @@ outputs:
...
+- contextPath: CarbonBlackEDR.Sensor.uninstall
+  description: when set, indicates sensor will be directed to uninstall on next checkin.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8.yml:1305]:
```suggestion
  description: When set, indicates that the sensor will be directed to uninstall on next check-in.
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2_6_8_README.md [hunk 9]
```diff
@@ -47,16 +53,20 @@ This playbook does not use any scripts.
...
-| CbResponse.Sensors.Isolated | Whether the sensor is isolated. | unknown |
 | MicrosoftATP.MachineAction.ID | The machine action ID. | string |
 | MicrosoftATP.IsolateList | The IDs of the machines that were isolated. | string |
 | MicrosoftATP.NonIsolateList | The IDs of the machines that will not be isolated. | string |
 | MicrosoftATP.IncorrectIDs | Incorrect device IDs entered. | string |
 | MicrosoftATP.IncorrectHostnames | Incorrect device host names entered. | string |
 | MicrosoftATP.IncorrectIPs | Incorrect device IPs entered. | string |
 | Core.Isolation.endpoint_id | The ID of the isolated endpoint. | string |
+| CarbonBlackEDR.Sensor | The sensor info. | unknown |
+| CarbonBlackEDR.Sensor.id | The sensor id of this sensor. | unknown |
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2_6_8_README.md:64]:
```suggestion
| CarbonBlackEDR.Sensor.id | The ID of this sensor. | unknown |
```

File: Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2_6_8_README.md [hunk 10]
```diff
@@ -47,16 +53,20 @@ This playbook does not use any scripts.
...
+| CarbonBlackEDR.Sensor.is_isolating | Boolean representing sensor-reported isolation status. | unknown |
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2_6_8_README.md:65]:
```suggestion
| CarbonBlackEDR.Sensor.is_isolating | Boolean representing the sensor-reported isolation status. | unknown |
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_72.md [hunk 11]
```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+
+##### Isolate Endpoint - Generic V2
+
+- Replaced sub-playbook `Block Endpoint - Carbon Black Response V2` with sub-playbook `Block Endpoint - Carbon Black Response V2.1`
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_72.md:12]:
```suggestion
- Replaced the **Block Endpoint - Carbon Black Response V2** sub-playbook with the **Block Endpoint - Carbon Black Response V2.1** sub-playbook.
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_72.md [hunk 12]
```diff
@@ -0,0 +1,8 @@
...
+- Removed the unnecessary outputs: *CbResponse.Sensors.CbSensorID*, *CbResponse.Sensors.Status*, *CbResponse.Sensors.Isolated*.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_72.md:13]:
```suggestion
- Removed the following unnecessary outputs:
   - *CbResponse.Sensors.CbSensorID*
   - *CbResponse.Sensors.Status*
   - *CbResponse.Sensors.Isolated*
```

File: Packs/CommonPlaybooks/ReleaseNotes/2_3_72.md [hunk 13]
```diff
@@ -0,0 +1,8 @@
...
+- Added new outputs: *CarbonBlackEDR.Sensor*, *CarbonBlackEDR.Sensor.id*, *CarbonBlackEDR.Sensor.id*, *CarbonBlackEDR.Sensor.status*.
```
Comment (ShirleyDenkberg) [inline Packs/CommonPlaybooks/ReleaseNotes/2_3_72.md:14]:
```suggestion
- Added the following new outputs: 
   - *CarbonBlackEDR.Sensor*
   - *CarbonBlackEDR.Sensor.id*
   - *CarbonBlackEDR.Sensor.id*
   - *CarbonBlackEDR.Sensor.status*
```

File: Packs/Carbon_Black_Enterprise_Response/Playbooks/playbook-Block_Endpoint_-_Carbon_Black_Response_README.md [hunk 14]
```diff
@@ -1,39 +1,48 @@
...
 | Hostname | The hostname to isolate. | ${Endpoint.Hostname} | Optional |
 
 ## Playbook Outputs
+
 ---
 
 | **Path** | **Description** | **Type** |
 | --- | --- | --- |
-| CbResponse.Sensors.CbSensorID | Carbon Black Response Sensors IDs that are isolated. | unknown |
+| CbResponse.Sensors.CbSensorID | Carbon Black Response Sensors ids that has been isolated. | unknown |
```
Comment (ShirleyDenkberg) [inline Packs/Carbon_Black_Enterprise_Response/Playbooks/playbook-Block_Endpoint_-_Carbon_Black_Response_README.md:38]:
```suggestion
| CbResponse.Sensors.CbSensorID | Carbon Black Response sensor IDs that have been isolated. | unknown |
```

**Comment by ShirleyDenkberg** [comment]
@OmriItzhak Please regenerate the playbook-Endpoint_Enrichment_-_Generic_v2.1_6_8_README.md after implementing my edits in the yml file.
@altmannyarden Doc review completed.

File: Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2_6_8.yml [hunk 15]
```diff
@@ -372,6 +328,50 @@ tasks:
...
+      {
+        "position": {
+          "x": 1770,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
```
Comment (altmannyarden) [inline Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2_6_8.yml:371]: should be true

Comment (OmriItzhak) [reply Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2_6_8.yml]: Done.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**